Header_CPP.cpp

```diff
@@ -0,0 +1,96 @@
+#include "Task II - Header.h"

```

Comment:
If you renamed `Task II - Header.h` to `Store.h`, then you would probably rename this file to `Store.cpp`, since it contains all the definitions of the methods for the `Store` class.